showcase/tests/integration/components/hds/flyout/index-test.js

```diff
@@ -113,6 +113,15 @@ module('Integration | Component | hds/flyout/index', function (hooks) {
     assert.dom('.hds-flyout__tagline').doesNotExist();
   });
 
+  test('it renders the title as an h1', async function (assert) {
```

Comment:
[suggestion]
```suggestion
  test('it renders the title as an h1 by default', async function (assert) {
```

Reply:
Yes. My suggestion was to make it more explicit in the title of the test. No biggie :)